dashboard_node:
  ros__parameters:
    # 토픽 구독 설정
    px4_local_pos_topic: "/fmu/out/vehicle_local_position"
    vehicle_odom_topic: "/model/X1/odometry"
    camera_image_topic: "/x500/image_proc"
    marker_detection_topic: "/marker_detections"
    drone_state_topic: "/drone/state"
    vehicle_state_topic: "/vehicle/state"
    mission_status_topic: "/mission/status"
    
    # TF 프레임 설정
    drone_frame_id: "x500_gimbal_0"
    vehicle_frame_id: "X1_asp"
    map_frame: "map"
    
    # 타이밍 설정
    check_timeout: 2.0
    node_timeout: 5.0


mission_control_node:
  ros__parameters:
    # TF 프레임 설정
    drone_frame_id: "x500_gimbal_0"
    vehicle_frame_id: "X1_asp"
    map_frame: "map"
    
    # 타이밍 설정
    check_timeout: 2.0


waypoint_mission_node:
  ros__parameters:
    # 드론 짐벌 카메라 프레임 설정
    gimbal_camera_frame: "x500_gimbal_0/camera_link"
    
    # 미션 진행 설정
    hover_duration: 2.0
    waypoint_reach_tolerance: 2.0


interactive_mission_node:
  ros__parameters:
    # 이륙 설정
    takeoff_altitude: 10.0
